Various fixups for the XRootD Standalone install doc (#673)
opensciencegrid/docs: docs/data/xrootd/install-standalone.md

1.  If you want to limit the sub-directories to serve under your configured `rootdir`,
    comment out the `all.export /` directive in
    `/etc/xrootd/config.d/90-osg-standalone-paths.cfg`,
    and add an `all.export` directive for each directory under `rootdir` that you wish to serve via XRootD.
    
    This is useful if you have a mixture of files under your `rootdir`, for example from multiple users,
    but only want to expose a subset of them to the world.
    
    For example, to serve the contents of `/data/store` and `/data/public` (with `rootdir` configured to `/data`):

        all.export /store/
        all.export /public/

    If you want to serve everything under your configured `rootdir`, you don't have to change anything.

    !!! note
        The directories specified this way are writable by default.
        Access controls should be managed via [authorization configuration](#configuring-authorization).

1. In `/etc/xrootd/config.d/10-common-site-local.cfg`, add a line to set the `resourcename` variable to the
   [resource name](/common/registration/#registering-resources) of your XRootD service.
   For example, the XRootD service registered at the

>    [University of Florida site](https://github.com/opensciencegrid/topology/blob/master/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)

Reviewer: ```suggestion
   [University of Florida site](https://github.com/opensciencegrid/topology/blob/b14218d6e9d9df013a42e4d8538b2eeea615514c/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)
```

Grossly long but let's use a link that'll guarantee a file that doesn't change.

To validate an XRootD installation, perform the following verification steps:

!!! note
    If you have configured authentication/authorization for XRootD,
    be sure you have given yourself the necessary permissions to run these tests.
    For example, if you are using a grid proxy,

>     make sure your DN is mapped to a user in `/etc/grid-security/grid-mapfile`,

Reviewer: ```suggestion
    make sure your DN is mapped to a user in [/etc/grid-security/grid-mapfile](/security/lcmaps-voms-authentication#mapping-users),
```

    and make sure you have a valid proxy on your local machine.

>     Also make sure that the Authfile on the XRootD server gives write access to the Unix user you will get mapped to.

Reviewer: ```suggestion
    Also, ensure that the [Authfile](/data/xrootd/xrootd-authorization#authorization-file) on the XRootD server gives write access to the Unix user you will get mapped to.
```
